### PR TITLE
add further bindings for ButtonDropdown

### PR DESCRIPTION
**Pull request by svengiebel**
I completed the bindings for the component `ButtonDropdown`.

Thanks for the repo 👍

### CODE REVIEW

File: src/BsReactstrap__ButtonDropdown.re [hunk 1]
```diff
@@ -1,20 +1,12 @@
-[@bs.module "reactstrap"] external buttonDropdown : ReasonReact.reactClass = "ButtonDropdown";
+[@bs.module "reactstrap"]
+external buttonDropdown : ReasonReact.reactClass = "ButtonDropdown";
 
 [@bs.obj]
-external makeProps : (
+external makeProps : (~isOpen: bool=?, ~toggle: 'a=?, unit) => _ = "";
```
Comment (liubko) [inline src/BsReactstrap__ButtonDropdown.re]: can you annotate `toggle`, please?
I guess it's smth like
`ReactEventRe.Form.t => unit`
or
`unit => unit`

I don't know if it accepts any arguments

Comment (svengiebel) [reply src/BsReactstrap__ButtonDropdown.re]: Seems like it passes the event right?

from reactstrap:
```
toggle(e) {
    if (this.props.disabled) {
      return e && e.preventDefault();
    }

    return this.props.toggle(e);
```

**Comment by svengiebel** [comment]
Seems like it passes the event right?

from reactstrap:
```
toggle(e) {
    if (this.props.disabled) {
      return e && e.preventDefault();
    }

    return this.props.toggle(e);
```

So use: `ReactEventRe.Mouse.t` or `ReactEventRe.Synthetic.t` ? Since it uses `preventDefault` it should be `ReactEventRe.Mouse.t` ?

**Comment by rusty-key** [comment]
@svengiebel, yep, it looks like it handles mouse click

**Review by rusty-key — APPROVED** [review]

**Comment by liubko** [comment]
published 0.2.2

**Pull request merged**